### DISCUSSION TITLE
Production Mode

### DISCUSSION THREAD

**Discussion by bm888** [tidal-music/discussions#65] [Q&A] [answered]
When can we apply for production mode? Thanks!

On the dashboard, I see:
"Production Mode Request - coming soon."

**Comment by stefanmoro** [comment, marked as answer]
Hey @bm888,

We are currently working hard on building out the API, and we will most likely also make some improvements to the existing endpoints before we remove the "Beta tag" on TIDAL Developer. We plan to enable the possibility to submit Production Mode requests approximately at the same time as we leave "Beta", something that will happen later this year. I unfortunately don't have a fixed date, but my hope is that it will happen in Q3.
